src/ARTED/control/control_ms.f90

```diff
@@ -63,14 +63,12 @@ subroutine tddft_maxwell_ms
     end if
 
 !====RT calculation============================
-    if ( .not. (restart_option == 'restart')) then
```

Comment:
Why do you remove this `if` statement? I added it to avoid incorrect initialization in PR #212.

Reply:
@shunsuke-sato san, it is not directly affect to this bugfix. Which bug tracking, I have exhaustively tested  if statements related to restarting. Then, I found Line:66 is doubly checks same condition defined in Line:47. 

It has not the origin of Issue #237, therefore I think both existing and removing are acceptable in this code.